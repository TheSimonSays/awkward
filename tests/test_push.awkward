let arr = [17];
print("arr len before " len(arr));
let x = pop(arr);
print("pop value x is " x);
print("x type is " typeof(x));
print("len arr after is " len(arr));
push(arr, 22);
print("arr after push 22 is "arr);
clear(arr);
print("arr after clear is " arr);
