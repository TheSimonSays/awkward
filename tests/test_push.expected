arr len before  1
pop value x is  17
x type is  int
len arr after is  0
arr after push 22 is  [22]
arr after clear is  []
